### PR TITLE
fix: mouse-wheel events were ignored

### PR DESCRIPTION
**Pull request by ldd**

### CODE REVIEW

File: src/input/pointer.js [hunk 1]
```diff
@@ -354,6 +354,17 @@
...
+                    case MOUSE_WHEEL:
+                        // event inside of bounds: trigger the MOUSE_WHEEL callback
+                        if (eventInBounds) {
+                            // trigger the corresponding callback
+                            if (triggerEvent(handlers, e.type, e, e.pointerId)) {
+                                handled = true;
+                                break;
+                            }
+                        }
+                        break;
```
Comment (parasyte) [inline src/input/pointer.js:356]: This looks identical to `POINTER_DOWN`. You should be able to add `case MOUSE_WHEEL:` above for a smaller patch.

Comment (parasyte) [inline src/input/pointer.js:356]: Or use `default:` here and delete the `POINTER_DOWN` case. I kind of prefer that, TBH. It may be more future-proof.

**Comment by ldd** [comment]
done. If I recall correctly, the original code was also future-proof

**Comment by parasyte** [comment]
Perfect! Thanks for debugging and patching. And yeah, the original code didn't use a switch statement for event dispatching. :smile:

**Pull request merged**